src/lib/containers/personal_access_token/AccessTokenCard.tsx

```diff
@@ -130,7 +125,7 @@ export const AccessTokenCard: React.FunctionComponent<AccessTokenCardProps> = ({
             }
           }}
         >
-          <FontAwesomeIcon icon={faTrash} aria-hidden="true" />
+          <IconSvg options={{ icon: 'delete' }} aria-hidden="true" />
```

Comment:
I don't think the aria-hidden gets passed along. IconSvg hould pass most props to its `span` wrapper.